doc/POINTS_COORDINATES.md

```diff
@@ -0,0 +1,169 @@
+# Points, tripoints, and coordinate systems
+
+## Coordinate systems
+
+CDDA uses a variety of coordinate systems for different purposes.  These differ
+by scale and origin.
+
+The most precise coordinates are **map square** (ms) coordinates.  These refer to
+the tiles you see normally when playing the game.
+
+Two origins for map square coordinates are common:
+* **Absolute** coordinates, which are a global system for the whole game,
+  relative to a fixed origin.
+* **Local** coordinates, which are relative to the corner of the current "reality
+  bubble", or `map` roughly centered on the avatar.  In local map square
+  coordinates, `x` and `y` values will both fall in the range [0,120).
+
+The next scale is **submap** (sm) coordinates.  One submap is 24x24
```

Comment:
12x12, I would think?  since an OMT is 2x2 submaps and 24x24 map squares.

Reply:
Whoops!  Fixed.